src/api/MainSolver.cc

```diff
@@ -95,34 +82,29 @@ bool MainSolver::pop()
         frames.pop();
         substitutions.pop();
         firstNotSimplifiedFrame = std::min(firstNotSimplifiedFrame, frames.frameCount());
-        if (not isLastFrameUnsat()) {
-            getSMTSolver().restoreOK();
-        }
+        if (not isLastFrameUnsat()) { getSMTSolver().restoreOK(); }
         return true;
     }
     return false;
 }
 
-PartitionManager & MainSolver::getPartitionManager() { return pmanager; }
-
-sstat MainSolver::getStatus() const { return status; }
-
-void MainSolver::insertFormula(PTRef fla)
-{
+void MainSolver::insertFormula(PTRef fla) {
```

Comment:
I think that format
```
Type foo(...)
{
...
}
```
is better, because when the function should catch an exception, then it looks like
```
Type foo(...)
try {
...
}
catch ...
```
In the former format, the `try` keyword may be difficult to spot, e.g. `Type foo(maaany params...) try {`.